app/src/main/java/org/mozilla/focus/widget/AnimatedProgressBar.java

```diff
@@ -297,6 +293,11 @@ public void onAnimationRepeat(Animator animator) {
 
     private void setVisibilityImmediately(int value) {
         super.setVisibility(value);
+        if (value == GONE) {
+            getProgressDrawable().setVisible(false, false);
+        } else {
```

Comment:
Not sure if we want to differentiate between Visible and Invisible here.

Reply:
we also want to stop the animation when Invisible. So let's keep it as is.